daemon/README.md

```diff
@@ -0,0 +1,5 @@
+Scoot Daemon Server.
+
+A server that serves the Scoot Deamon API.
+
+For now this is across Unix Domain Sockets, but this detail is hidden behind the interfaces.
```

Comment:
I'd like to keep this the minimal changes for the rename, so I added TODO's here and in locate.go.